parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -68,113 +128,163 @@ mod bridge_hub_rococo_tests {
+			Runtime,
+			XcmConfig,
+			WithBridgeHubWococoMessagesInstance,
+		>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			1000,
+			Box::new(|runtime_event_encoded: Vec<u8>| {
+				match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+					Ok(RuntimeEvent::BridgeWococoMessages(event)) => Some(event),
+					_ => None,
+				}
+			}),
+			|| ExportMessage { network: Wococo, destination: X1(Parachain(1234)), xcm: Xcm(vec![]) },
+			bridge_hub_rococo_config::DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO
+		)
+	}
 
-	bridge_hub_test_utils::include_message_dispatch_routing_works!(
-		Runtime,
-		XcmConfig,
-		ParachainSystem,
-		WithBridgeHubWococoMessagesInstance,
-		RelayNetwork,
-		bridge_hub_rococo_config::WococoGlobalConsensusNetwork,
-		bridge_hub_test_utils::CollatorSessionKeys::new(
-			AccountId::from(ALICE),
-			AccountId::from(ALICE),
-			SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) }
-		),
-		bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
-		1000,
-		Box::new(|runtime_event_encoded: Vec<u8>| {
-			match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
-				Ok(RuntimeEvent::ParachainSystem(event)) => Some(event),
-				_ => None,
-			}
-		}),
-		Box::new(|runtime_event_encoded: Vec<u8>| {
-			match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
-				Ok(RuntimeEvent::XcmpQueue(event)) => Some(event),
-				_ => None,
-			}
-		}),
-		bridge_hub_rococo_config::DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO
-	);
+	#[test]
+	fn message_dispatch_routing_works() {
+		bridge_hub_test_utils::test_cases::message_dispatch_routing_works::<
+			Runtime,
+			XcmConfig,
+			ParachainSystem,
+			WithBridgeHubWococoMessagesInstance,
+			RelayNetwork,
+			bridge_hub_rococo_config::WococoGlobalConsensusNetwork,
+		>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			1000,
+			Box::new(|runtime_event_encoded: Vec<u8>| {
+				match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+					Ok(RuntimeEvent::ParachainSystem(event)) => Some(event),
+					_ => None,
+				}
+			}),
+			Box::new(|runtime_event_encoded: Vec<u8>| {
+				match RuntimeEvent::decode(&mut &runtime_event_encoded[..]) {
+					Ok(RuntimeEvent::XcmpQueue(event)) => Some(event),
+					_ => None,
+				}
+			}),
+			bridge_hub_rococo_config::DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO,
+		)
+	}
+
+	#[test]
+	fn relayed_incoming_message_works() {
+		bridge_hub_test_utils::test_cases::relayed_incoming_message_works::<
+			Runtime,
+			XcmConfig,
+			ParachainSystem,
+			BridgeGrandpaWococoInstance,
+			BridgeParachainWococoInstance,
+			WithBridgeHubWococoMessagesInstance,
+			WithBridgeHubWococoMessageBridge,
+		>(
+			collator_session_keys(),
+			bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			bp_bridge_hub_wococo::BRIDGE_HUB_WOCOCO_PARACHAIN_ID,
+			1000,
+			Rococo,
+			LaneId([0, 0, 0, 1]),
```

Comment:
There are couple of places where we use `LaneId([0, 0, 0, 1])` directly. Is there a reason not to use `DEFAULT_XCM_LANE_*` everywhere? IIUC we'll be using the same code in Kusama and Polkadot runtime && it'll be easier to deal with constants when copy-pasting. Also `1000` above - maybe worth to add e.g. `SIBLING_PARACHAIN` constant too.